### PR TITLE
[WiP] Add a to_hdf method to the MontecarloRunner.

### PR DESCRIPTION
**Pull request by ftsamis**
For now this is a very simple `to_hdf` method which only stores `output_nu` and `output_energy`.

### CODE REVIEW

File: tardis/montecarlo/base.py [hunk 1]
```diff
@@ -220,3 +222,44 @@ def calculate_f_nu(self, frequency):
 
     def calculate_f_lambda(self, wavelength):
         pass
+
+    def property_to_hdf(self, property):
```
Comment (wkerzendorf) [inline tardis/montecarlo/base.py]: I would make this static and just use the value

Comment (wkerzendorf) [inline tardis/montecarlo/base.py]: nevermind

File: tardis/montecarlo/base.py [hunk 2]
```diff
@@ -220,3 +222,44 @@ def calculate_f_nu(self, frequency):
...
+        -------
+            : None
+
+        """
+        try:
+            hdf_store = pd.HDFStore(path_or_buf)
+        except TypeError:
+            hdf_store = path_or_buf
+
+        properties = ['output_nu', 'output_energy']
```
Comment (wkerzendorf) [inline tardis/montecarlo/base.py]: okay the properties that we want to store are `output_nu`, `output_energy`, `nu_bar_estimator`, `j_estimator`

Comment (ftsamis) [inline tardis/montecarlo/base.py]: Added `nu_bar_estimator`, `j_estimator`

File: tardis/montecarlo/base.py (hunk 1 above)
Comment (ftsamis) [inline tardis/montecarlo/base.py]: This needs some kind of change anyway. It does the same thing as `BasePlasmaProperty._get_output_to_hdf`. I'll probably introduce a similar method to `util` and use them everywhere needed.

Comment (wkerzendorf) [inline tardis/montecarlo/base.py]: don't let it live here. It's fine.

**Comment by ftsamis** [comment]
Closing this in favour of #585

**Pull request closed without merging**